css/ampstart-base/base-page.css

```diff
@@ -15,7 +15,22 @@
  */
 @import 'base-page-vars';
 
-* {
+amp-sidebar,
+div,
+section,
+input,
+textarea,
+select,
+label,
+button,
+.btn,
+.border,
+.border-top,
+.border-right,
+.border-bottom,
+.border-left,
+.ampstart-btn,
+.ampstart-input {
   box-sizing: border-box;
```

Comment:
Agreed, especially about overriding bass classes. I got a bit confused about the specificity when I was playing with the dev tools. Could we get away with just reseting some likely to be bordered type selectors? People can add what isn't covered for their specific use.